todo_app/templates/index.html

```diff
@@ -1,18 +1,67 @@
 {% extends "layout.html" %}
 {% block title %}To-Do App{% endblock %}
-
 {% block content %}
   <div class="jumbotron">
     <h1 class="display-4">To-Do App</h1>
     <p class="lead">Just another to-do app.</p>
   </div>
-
   <div class="row">
     <div class="col-md-12">
-      <h2>Items</h2>
+      <h3>Items</h3>
       <ul class="list-group mb-4">
-        <!-- List the todo items here -->
+        <span class="border border-primary">
+          <table class="table table-sm table-striped table-hover table-bordered">
+              <tr class="table-primary">
+                <th>Id</th>
+                <th>Item</th>
+                <th>Status</th>
+                <th>Action</th>
+              </tr>
+            <tbody>
+              {% for item in items%}
+              <tr>
+                <td>{{item.id}}</td>
+                <td>{{item.title}}</td>
+                <td>{{item.status}}</td>
+                <td>
+                  <div class="btn-toolbar" role="toolbar" aria-label="Toolbar with button groups"></div>
+                    <div class="btn-group mr-1" role="group" aria-label="in progress">
+                      <form action="http://localhost:5000/inprogress" method="POST">
```

Comment:
Note that this works but means you'll never be able to deploy the system as people will need the code running on their machines! If we remove the domain from the URL it will default to the domain it's running on so will be more flexible 🙂 

```suggestion
                      <form action="/inprogress" method="POST">
```

Reply:
Incorporated suggestion